sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.9.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.9.0 (2025-04-08)
 
 ### Other Changes
 
+- Update README for clearer guidance.
```

Comment:
can this be clearer? :p

Add a link to the commit and summarize the change in a line perhaps.

Reply:
Updated